README.rst

```diff
@@ -18,7 +18,13 @@ Setup
 -----
 
 1. Clone this repository
-2. Export the environment variables ``OSC_USER`` and ``OSC_PASSWORD`` containing
+   git clone https://github.com/dcermak/scm-staging
+
+2. Install required library
+   cd scm-staging
+   pip3 install -e ./
```

Comment:
This project uses poetry, so I would advise to use that instead of pip
```suggestion
2. Install dependencies:
   cd scm-staging
   poetry install
```
```

Reply:
Ah sure I see how you do it now. 
I did not use poetry before, is it possible to add something like the follow? 

   python3.11 -m venv ./
   pip3.11 install poetry
   poetry install

Reply:
That's possible, but definitely not recommended. Poetry is a tool to manage virtual environments and putting it into a virtual environment could lead to trouble. There are other recommended ways how to install poetry, I would suggest to use pipx: https://python-poetry.org/docs/#installing-with-pipx

Reply:
Sure, I don't have experience with poetry sorry about that. I will take a look at poetry and pipx. Thanks